README.md

```diff
@@ -30,45 +35,77 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Data Science](https://medium.com/ibm-watson): Systems and scientific methods to analyze structured and unstructured data in order to extract knowledge and insights.
 * [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
 
-# Watch the Video
-
 # Steps
 
-## Run locally
+Either Run locally:
 
 1. [Clone the repo](#1-clone-the-repo)
-1. [Run the notebook](#2-run-the-notebook)
+2. [Run Jupyter Notebooks](#2-run-jupyter-notebooks)
+
+or in Watson Studio:
+
+3. [Sign up for the Watson Studio](#3-sign-up-for-watson-studio)
+4. [Create the notebook](#4-create-the-notebook)
+
+then:
+
+5. [Run the Notebook](#5-run-the-notebook)
+
+## Run locally
 
 ### 1. Clone the repo
 
-Clone the `xxxxxxxx` locally. In a terminal, run:
+Clone the `https://github.com/IBM/ensure-loan-fairness-aif360` locally. In a terminal, run:
 
 ```
-$ git clone https://github.com/IBM/xxxxxxxxxx
+$ git clone https://github.com/IBM/ensure-loan-fairness-aif360
 ```
 
-### 2. Run the notebook
+### 2. Run Jupyter Notebooks
 
 The code included in this Code Pattern runs in a Jupyter Notebook.
 
-* Start your Jupyter Notebooks. Starting in your `xxxxxxxxx` cloned repo directory will help you find the notebook and the output as described below. Jupyter Notebooks will open in your browser.
+* Start your Jupyter Notebooks. Starting in your `ensure-load-fairness-aif360` cloned repo directory will help you find the notebook and the output as described below. Jupyter Notebooks will open in your browser.
 
    ```
-   cd xxxxxxxxx
+   cd ensure-load-fairness-aif360
    jupyter notebook
    ```
 
-* Navigate to the `notebooks` directory and open the notebook file named `xxxxx.ipynb` by clicking on it.
+* Navigate to the `notebooks` directory and open the notebook file named `credit_scoring.ipynb` by clicking on it.
+
+## Run in Watson Studio
+
+## 3. Sign up for Watson Studio
+
+Sign up for IBM's [Watson Studio](https://dataplatform.ibm.com). By creating a project in Watson Studio a free tier ``Object Storage`` service will be created in your IBM Cloud account. Take note of your service names as you will need to select them in the following steps.
```

Comment:
I don't think the double backticks do anything that single ones don't do (right?)